test/Microsoft.DotNet.Web.Authentication.Tests/Controllers/AnonymousAttributeController.cs

```diff
@@ -7,9 +7,9 @@
 namespace Microsoft.DotNet.Web.Authentication.Tests.Controllers;
 
 [Route("test-auth/anonymous")]
-[AllowAnonymous]
 public class AnonymousAttributeController : ControllerBase
 {
+    [AllowAnonymous]
```

Comment:
Why this change? Was this CodeQL also?

Reply:
Yup, they didn't want Authorize methods to inherit Anonymous.